### DISCUSSION TITLE
如何停掉图片和表格的转换

### DISCUSSION THREAD

**Discussion by studylittle** [magicpdf/Magic-PDF#723] [Q&A]
如何停掉图片和表格的转换，识别，使得生成的md文档没有图片或者表格，这样转换速度也加快

**Comment by studylittle** [comment]
可以在这里过滤一下返回新的layout_line

**Reply by studylittle** [reply]
pdf_extract_kit.py这个文件中返回的layout_line返回值过滤，可以查看        for line in result:
            line.pop("img")
            """
            为paddle输出适配type no.    
            title: 0 # 标题
            text: 1 # 文本
            header: 2 # abandon
            footer: 2 # abandon
            reference: 1 # 文本 or abandon
            equation: 8 # 行间公式 block
            equation: 14 # 行间公式 text
            figure: 3 # 图片
            figure_caption: 4 # 图片描述
            table: 5 # 表格
            table_caption: 6 # 表格描述
            """
            if line["type"] == "title":
                line["category_id"] = 0
            elif line["type"] in ["text", "reference"]:
                line["category_id"] = 1
            elif line["type"] == "figure":
                line["category_id"] = 3
            elif line["type"] == "figure_caption":
                line["category_id"] = 4
            elif line["type"] == "table":
                line["category_id"] = 5
            elif line["type"] == "table_caption":
                line["category_id"] = 6
            elif line["type"] == "equation":
                line["category_id"] = 8
            elif line["type"] in ["header", "footer"]:
                line["category_id"] = 2
            else:
                logger.warning(f"unknown type: {line['type']}")

            # 兼容不输出score的paddleocr版本

**Comment by papayalove** [comment]
可以在magic-pdf.json配置文件中设置表格的开关为false
